apps-rendering/src/client/article.ts

```diff
@@ -475,3 +474,4 @@ initAudioAtoms();
 hydrateAtoms();
 richLinks();
 hydrateClickToView();
+reportNativeElementPositionChanges();
```

Comment:
What do you think about adding a comment here? I think it might be useful to explain we want `reportNativeElementPositionChanges()` to come last and possibly link back to this PR

Reply:
A good suggestion which I've taken